docs/getting-started/example-deployment/project-setup.md

```diff
@@ -0,0 +1,231 @@
+    │   ├── gaffer
+    │   │   ├── application.properties #(1)!
+    │   │   ├── data                   #(2)!
+    │   │   │   ├── neo4jExport.csv
+    │   │   ├── graph
+    │   │   │   └── graphConfig.json #(3)!
+    │   │   ├── schema
+    │   │   │   ├── elements.json #(4)!
+    │   │   │   └── types.json    #(5)!
+    │   │   └── store
+    │   │       ├── operationsDeclarations.json #(6)!
+    │   │       └── store.properties            #(7)!
+    │   └── hdfs
+    │       ├── core-site.xml
+    │       ├── hdfs-site.xml
+    │       └── log4j.properties
+    └── docker-compose.yaml #(8)!
+    ```
+
+    1. Properties file that generally sets the file locations of other Gaffer
+        configs e.g. schemas (note these are the absolute paths inside the
+        container).
+    2. Any data files, e.g. CSV, to be made available to the Gaffer container.
+    3. The main graph config file to set various properties of the overall graph.
+    4. This file holds the schema outlining the elements in the graph, e.g. the
+    nodes (aka entities) and edges.
+    5. This file defines the different data types in the graph and how they are
+    serialised to Java classes.
+    6. Config file for additional Gaffer operations and set the class to handle
+    them on the store.
+    7. The General store properties, sets up what store to use and any additional
+    configuration.
+    8. This file controls which containers will be started up and the configuration
+    of them to ensure correct ports and files are available.
+
+All the files in the `config/accumulo/` and `config/hdfs/` directories will be copied directly from
+the two locations in the Gaffer docker repo,
+[here](https://github.com/gchq/gaffer-docker/tree/develop/docker/accumulo/conf-2.0.1) and
+[here](https://github.com/gchq/gaffer-docker/tree/develop/docker/hdfs/conf). The configuration of
+these are out of scope of this example but are covered in other sections of the documentation. The
+main focus of this guide will be on the configuration files under the `config/gaffer/` directory.
+
+## Configuration Files
+
+There's a full [break down of Gaffer schema files on the next page](./writing-the-schema.md), this
+section will instead cover the smaller additional config files that go along side the main Gaffer
+schema to tweak other aspects of the graph. The location of these files will need to be volume
+mounted into the container for them to be included in the deployment which is covered in more detail
+[later in the guide](./running-the-deployment.md).
+
+!!! note
+    Many of these files have defaults already in the standard `gaffer-rest` container image, but its
+    useful to still include them in the project to allow easy configuration.
+
+### Application Properties
+
+This is probably the simplest configuration file in the Gaffer deployment. In general it borrows a
+concept from [Spring
+Boot](https://www.tutorialspoint.com/spring_boot/spring_boot_application_properties.htm) to allow
```

Comment:
Properties files are not specific to Spring, although the context root change does apply to Spring.